src/closure/closureWithArrayClosures.groovy

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env groovy
+package closure
+
+
```

Comment:
Where is an example of your own closures?

Where is an example of using REST API?